scripts/purge.ps1

```diff
@@ -4,9 +4,7 @@
 # to find all processes of a particular name which have file locks on our build (project) directory.
 # We can then attempt to kill each of these processes if any exist.
 
-$ScriptLocation = $MyInvocation.MyCommand.Path
-$ScriptDir = Split-Path $ScriptLocation
-cd "$ScriptDir/../.."
+cd $projectRoot
```

Comment:
Small nit - can we add usage to the comment above since it takes an argument now :)

Reply:
good shout 👍